src/video/windows/SDL_windowsevents.c

```diff
@@ -1284,6 +1289,25 @@ WIN_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
         }
         break;
 
+#if HAVE_TPCSHRD_H
+
+    case WM_TABLET_QUERYSYSTEMGESTURESTATUS:
+        // See https://msdn.microsoft.com/en-us/library/windows/desktop/bb969148(v=vs.85).aspx .
```

Comment:
C-style /**/ comments maybe?

Reply:
That's fine, but there are two other C++-style comments in that file (lines 309 and 1313, in the main branch).

Reply:
Let's just not add more to it.

Reply:
OK, done.